shopfloor_mobile/static/wms/src/scenario/single_pack_transfer.js

```diff
@@ -76,6 +76,11 @@ const SinglePackTransfer = {
                     :card_color="utils.colors.color_for('screen_step_done')"
                 >
                     <template v-slot:after_details>
+                        <v-card-subtitle>
+                            <span>
+                                {{ state.data.picking.weight.toFixed(2)}} {{state.data.weight_uom}}
```

Comment:
The picking weight is an estimation based on product data. And there could be more than just the pack in the picking.

I would display the package `pack_weight` or, if empty, the `estimated_pack_weight_kg` converted to the package weight_uom_id

Maybe add the `estimated_pack_weight` on stock-logistics-workflow/stock_quant_package_dimension

Reply:
I see the exact opposite was explicitly made some time ago, to make sure `estimated_pack_weight` was always in kg:

![image](https://github.com/OCA/wms/assets/77412816/fb0537d3-f451-4e90-8c6d-db67b65d2974)

Do we wanna change that back?

I've modified the scenario to use the uom if there's a `pack_weight`, and otherwise use `estimated_pack_weight_kg` and display it as kg. What do you think?

Reply:
Sounds great. Indeed if there is no pack_weight, there is likely no uom, so we better display estimated_pack_weight_kg in kg